metricbeat/module/elasticsearch/index_summary/data.go

```diff
@@ -47,6 +47,47 @@ var (
 					"bytes": c.Int("memory_in_bytes"),
 				},
 			}),
+			"indexing": c.Dict("indexing", s.Schema{
```

Comment:
There is some redundancy between the schema definition for `"indexing"` here and also further below. You may consider extract the definition to a variable and referencing it from both places.

Reply:
I grouped all the redundant fields. Thanks for reminding me this! :smile: